helix-core/src/main/java/org/apache/helix/model/InstanceConfig.java

```diff
@@ -327,7 +314,7 @@ public String getInstanceDisabledReason() {
    */
   public String getInstanceDisabledType() {
     if (getInstanceEnabled()) {
-      return InstanceConstants.InstanceDisabledType.INSTANCE_NOT_DISABLED.toString();
+      return InstanceConstants.INSTANCE_NOT_DISABLED;
```

Comment:
So we changed this in last PR for string?

Reply:
Yes. It would be confusing for user to see "INSTANCE_NOT_DISABLED" in InstanceDisabledType. I use to a separate static String to represent not enabled instance.